platform-support/repo/nebulatest/platform/1.0.0/platform-1.0.0.pom

```diff
@@ -8,6 +8,24 @@
   <packaging>pom</packaging>
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-framework-bom</artifactId>
+        <version>5.0.6.RELEASE</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
```

Comment:
This exclusion would have no impact in Gradle. The support for `import` is not new, and has always been part of Gradle's support for POMs.

If it can be shown that Maven supports `exclusions` on dependencies with `import` scope, we could look at changing this.

Reply:
@bigdaz It looks like Maven is currently adding support for the import exclusions. It might get released with Maven 3.6, but it also looks like this feature hasn't had a lot of attention: https://issues.apache.org/jira/browse/MNG-5600

Reply:
Thanks @jawaff. I think we'll hold off making any changes around this until Maven 3.6 is released with this support.